src/com/company/InMemoryNameRepository.java

```diff
@@ -0,0 +1,24 @@
+package com.company;
```

Comment:
An excellent name for this :)

Reply:
I would love to take credit but that was Sol